libs/dev-dock/frontend/src/dev_dock.tsx

```diff
@@ -212,6 +213,7 @@ const SmartCardMocksDisabledMessage = styled.div`
 `;
 
 const ROLES = [
+  'vendor',
```

Comment:
You can see here what the dev dock looks like with one more card option:

<img width="1512" alt="dev-dock" src="https://github.com/votingworks/vxsuite/assets/12616928/0e169dd3-1622-4aac-a3b2-7f0206932dac">